crates/ruff_linter/src/rules/pyflakes/rules/unused_import.rs

```diff
@@ -330,7 +330,7 @@ pub(crate) fn unused_import(checker: &Checker, scope: &Scope, diagnostics: &mut
                     fix_by_reexporting(
                         checker,
                         import_statement,
-                        &to_reexport.iter().map(|(b, _)| b).collect::<Vec<_>>(),
+                        &mut to_reexport.iter().map(|(b, _)| b).collect::<Vec<_>>(),
```

Comment:
Maybe we should just pass in the `Vec` here, so the ownership is clearer? Then it can be `mut imports: Vec<&ImportBinding>` and callers don't need to worry about the mutation.

Reply:
Tweaking this quickly so I can include in the release.

Reply:
Ok! Thanks for explaining